src/frontends/lean/pp.cpp

```diff
@@ -1847,9 +1956,12 @@ auto pretty_fn::pp(expr const & e, bool ignore_hide) -> result {
             return pp_sep(e);
         if (is_set_of(e))
             return pp_set_of(e);
-        buffer<expr> elems;
-        if (is_explicit_collection(e, elems))
+        buffer<subexpr> elems;
+        if (is_explicit_collection(e, elems)) {
+            std::reverse(elems.begin(), elems.end());
```

Comment:
This seems like a change in behavior?

Reply:
ah ok

Reply:
Relevant issue is #153. I think this reverse is necessary but will check again.

Reply:
The reverse is necessary, but can be refactored inside is_explicit_collection if desired.

Reply:
I don't really care where it is reversed, as long as it doesn't break #153.